examples/basic/main.tf

```diff
@@ -0,0 +1,75 @@
+terraform {
```

Comment:
in our other mcaf modules we keep this seperate in an terraform.tf file which is the same as the main modules terraform.tf file